File: milestone 2/python_modules/module00/ex2/ft_harvest_total.cpp
#include <iostream>

void	ft_harvest_total()
{
	int day = 0, total = 0;
	std::cout << "Day 1 harvest: ";
	std::cin >> day;
	total += day;
	std::cout << "Day 2 harvest: ";
	std::cin >> day;
	total += day;
	std::cout << "Day 3 harvest: ";
	std::cin >> day;
	total += day;
	std::cout << "Total harvest: " << total;
	std::cout << std::endl;
}

int		main(void)
{
	ft_harvest_total();
	return 0;
}

File: milestone 2/python_modules/module00/ex2/ft_harvest_total.java
import java.util.Scanner;

public class ft_harvest_total {
    public static void main(String[] args)
    {
        System.out.print("Day 1 harvest: ");
        Scanner scan = new Scanner(System.in);
        int first = scan.nextInt();
        System.out.print("Day 2 harvest: ");
        Scanner scan2 = new Scanner(System.in);
        int second = scan2.nextInt();
        System.out.print("Day 3 harvest: ");
        Scanner scan3 = new Scanner(System.in);
        int third = scan3.nextInt();
        int total = (first + second + third);
        System.out.println("Total harvest: " + total);
    }
}